File: Beginner - C++/1010.cpp
#include <iomanip>
#include <iostream>

int main ()
{
    double code1, units1, value1, code2, units2, value2;
    std::cin >> code1 >> units1 >> value1;
    std::cin >> code2 >> units2 >> value2;
    std::cout << std::fixed << std::setprecision(2) << "VALOR A PAGAR: R$ " << units1*value1+units2*value2 << std::endl;

    return 0;
}

File: Para Lembrar/1010.py
def total():
    prod1 = str(input())
    uni1 = float(input())
    value1 = float(input())
    prod2 = str(input())
    uni2 = float(input())
    value2 = float(input())
    total = uni1*value1+uni2*value2

    print ("VALOR A PAGAR: R$ %0.2f" %total)

total()